protected/tests/unit/helpers/ArrayHelperTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class arrayHelperTest extends CTestCase{
```

Comment:
Does it matter that the casing of the class name doesn't match the name of the filer? I would have thought phpunit would baulk at that

Reply:
Probably doesn't, but it should be consistent regardless.